docs/audio.rst

```diff
@@ -17,14 +17,31 @@ There are three different kinds of audio sources that can be played using the
 
 1. `Built in sounds <#built-in-sounds-v2>`_ (**V2**),
```

Comment:
Just above this list of 5 things we say it's a list of 3 things!

Reply:
Thanks Matt! Resolved in b129d161192a640b0d034e9071e3b94af6731601.